specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/examples/SuccessfulAnalyzeConversationTextJobsTaskStatusRequest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "displayName": "Analyze chat",
+  "createdDateTime": "2022-04-01T15:00:45Z",
+  "expirationDateTime": "2022-04-02T15:00:45Z",
+  "jobId": "3e9e8518-492f-47f9-abd1-9a7468231086",
+  "lastUpdateDateTime": "2022-04-01T15:00:49Z ",
+  "status": "succeeded",
+  "tasks": {
+    "completed": 1,
+    "failed": 0,
+    "inProgress": 0,
+    "total": 1,
+    "items": [
+      {
+        "kind": "IssueResolutionSummarizationResults",
+        "lastUpdateDateTime": "2022-04-01T15:00:49Z",
+        "taskName": "analyze 1",
+        "status": "succeeded",
+        "results": {
+          "conversations": [
+            {
+              "id": "20220101meeting",
+              "summaries": [
+                {
+                  "aspect": "issue",
+                  "text": "Customer wants to cancel his subscription. Customer doesn't know how"
+                },
+                {
+                  "aspect": "resolution",
+                  "text": "pii Auto-Fortify. \npii Gamertag.\npii E-mail.\nCanceled customer's subscription"
+                }
+              ],
+              "warnings": [],
+              "statistics": {
+                "charactersCount": 48,
+                "transactionsCount": 1
+              }
+            }
+          ],
+          "errors": [],
+          "modelVersion": "2020-04-01"
+        }
+      }
+    ]
+  },
+  "@nextLink": "<resource-endpoint>/language/analyze-conversation/jobs/3e9e8518-492f-47f9-abd1-9a7468231086?$skip=10&$top=10"
```

Comment:
We no longer use the "@" symbol in the representation of `nextLink`.  This was recommended to be dropped in previous API versions in the Language API as well.